src/lib/authMgr.js

```diff
@@ -1,55 +1,78 @@
 import { decodeToken, TokenVerifier } from 'jsontokens'
+import { verifyJWT } from 'did-jwt'
 
 class AuthMgr {
 
     constructor() {
         this.nisabaPub=null
+        this.allowedIss=null
     }
     
     isSecretsSet(){
-        return (this.nisabaPub !== null);
+        return (this.nisabaPub !== null && this.allowedIss !== null);
     }
     
     setSecrets(secrets){
         this.nisabaPub=secrets.NISABA_PUBKEY;
+        this.allowedIss=secrets.ALLOWED_ISS.split(',');
     }
-    
-    async verifyNisaba(event){
-        if(!event.headers) throw('no headers')
-        if(!event.headers['Authorization']) throw('no Authorization Header')    
-        if(!this.nisabaPub) throw('nisabaPub not set')   
-        
+
+    async verifyFuelToken(event){
+        if(!event.headers) throw Error('no headers')
+        if(!event.headers['Authorization']) throw Error('no Authorization Header')  
+
         let authHead=event.headers['Authorization']
 
         let parts = authHead.split(' ')
         if (parts.length !== 2) {
-            throw('Format is Authorization: Bearer [token]')
+            throw Error('Format is Authorization: Bearer [token]')
         }
         let scheme = parts[0];
         if (scheme !== 'Bearer') {
-            throw('Format is Authorization: Bearer [token]')
+            throw Error('Format is Authorization: Bearer [token]')
         }
 
+        const token=parts[1];
         let dtoken
         try {
-            dtoken = decodeToken(parts[1])
+            dtoken = decodeToken(token)
```

Comment:
we should rely on `did-jwt` to create, decode and verify tokens IMO

Reply:
The problem is that to support the "old" token, we need to decode our self first